### PR TITLE
Add cost models to openapi.json.

### PR DESCRIPTION
**Pull request by werwty**
- add /costmodels endpoint and remove /rates endpoints

https://github.com/project-koku/koku/issues/974

### CODE REVIEW

**Comment by werwty** [comment]
Demo that the CLI generated from this spec works: https://asciinema.org/a/4OEwPb4R2UzE6gUdQmighTzfX

File: docs/source/specs/openapi.json [hunk 1]
```diff
@@ -705,31 +720,41 @@
...
+                                    "$ref": "#/components/schemas/Error"
+                                }
+                            }
+                        }
+                    },
                     "404": {
                         "description": "Not Found",
                         "content": {
-                            "application/json": {
+                            "*/*": {
```
Comment (infinitewarp) [inline docs/source/specs/openapi.json]: Do we ever (under normal circumstances) return a response without json?

Comment (werwty) [reply docs/source/specs/openapi.json]: I don't _think_ so. Koku's exception handler encodes errors as dicts: https://github.com/project-koku/koku/blob/master/koku/api/common/exception_handler.py#L67

And by default uses the json renderer: https://github.com/project-koku/koku/blob/master/koku/koku/settings.py#L260

The content type for errors in the openapi.json is kinda inconsistent. It's mostly `application/json` but there's a couple of `*/*`. I'm not sure if that's an oversight, or if I'm missing something...

Comment (adberglund) [reply docs/source/specs/openapi.json]: I think it would be safe to go with `application/json`

Comment (werwty) [reply docs/source/specs/openapi.json]: It's `application/json`
```
> GET /api/cost-management/v1/costmodels/34b2b968-6181-4104-b183-6d47d8243e95/ HTTP/1.1
> Host: 0.0.0.0:8000
> User-Agent: curl/7.61.1
> Accept: */*
> 
< HTTP/1.1 404 Not Found
< Date: Tue, 20 Aug 2019 18:33:49 GMT
< Server: WSGIServer/0.2 CPython/3.6.3
< Content-Type: application/json
< Vary: Accept, Origin
< Allow: GET, PUT, PATCH, DELETE, HEAD, OPTIONS
< X-Frame-Options: SAMEORIGIN
< Content-Length: 67
< 
{ [67 bytes data]
100    67  100    67    0     0   3722      0 --:--:-- --:--:-- --:--:--  3722
* Connection #0 to host 0.0.0.0 left intact
{
  "errors": [
    {
      "detail": "Not found.",
      "source": "detail",
      "status": 404
    }
  ]
}
```

**Pull request merged**